package.json

```diff
@@ -31,7 +31,8 @@
         "got": "^11.8.6",
         "mqtt": "^4.3.7",
         "semver": "^7.3.8",
-        "yaml": "^2.1.1"
+        "yaml": "^2.1.1",
+        "websocket": "^1.0.34"
```

Comment:
We should be using the maintained `ws` module - not `websocket` that hasn't had an update in 2 years.